Src/Persistence/Migrations/Logging.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace TeamProject.Persistence.Migrations
+{
+    public partial class Logging : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
+                "Message",
+                "Logs",
+                maxLength: 512,
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(512)",
+                oldMaxLength: 512,
+                oldNullable: true);
+
+            migrationBuilder.AlterColumn<string>(
+                "Level",
+                "Logs",
+                maxLength: 64,
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(64)",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define a constant instead of using this literal 'nvarchar(64)' 4 times.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5600534)